specification/hybridcompute/resource-manager/Microsoft.HybridCompute/preview/2024-05-20-preview/HybridCompute.json

```diff
@@ -2862,6 +2862,26 @@
             "name": "PatchModeTypes",
             "modelAsString": true
           }
+        },
+        "enableHotpatching": {
+          "type": "boolean",
+          "description": "Captures the hotpatch capability enrollment intent of the customers, which enables customers to patch their Windows machines without requiring a reboot."
```

Comment:
FYI, the CRP description is like: "Enables customers to patch their Azure VMs without requiring a reboot. For enableHotpatching, the 'provisionVMAgent' must be set to true and 'patchMode' must be set to 'AutomaticByPlatform'."

Reply:
Any action needed for the comment above?

Reply:
No, this is just a reference (for reviewers) to the same property in the Compute RP. We don't need to do anything here.